### PR TITLE
feat(movies): migration to add name column to movies

### PR DESCRIPTION
**Pull request by pallavi**
What: Adds `name` column to `movies` table and makes `title` nullable

### CODE REVIEW

File: db/migrations/20170825111648_add_name_to_movies.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = (Knex, Promise) => {
+  return Knex.schema.table('movies', (table) => {
+    table.text('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title SET NOT NULL');
```
Comment (robinjoseph08) [inline db/migrations/20170825111648_add_name_to_movies.js]: isn't the `title`column already `not null`?

File: db/migrations/20170825111648_add_name_to_movies.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  });
+};
+
+exports.down = (Knex, Promise) => {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title SET NOT NULL');
```
Comment (robinjoseph08) [inline db/migrations/20170825111648_add_name_to_movies.js:17]: looks like this is the same as the the `up`. is that intended?

File: db/migrations/20170825111648_add_name_to_movies.js (hunk 1 above)
Comment (pallavi) [reply db/migrations/20170825111648_add_name_to_movies.js]: oops meant to drop the `not null` constraint

**Review by robinjoseph08 — APPROVED** [review]
LGTM!

**Pull request merged**